.github/workflows/update-js.yml

```diff
@@ -13,9 +13,9 @@ on:
           - 'prev-major-curr-minor'
           - 'curr-major-curr-minor'
           - 'curr-major-next-minor'
-  # At 4:20 AM UTC, on day 1, 2, and 3 of the month, only in February and August
+  # At 4:20 AM UTC, on day 1 of the month, only in February and August
   schedule:
-    - cron: '20 4 1,2,3 2,8 *'
```

Comment:
This change is correct - refer to https://github.com/silverstripe/silverstripe-advancedworkflow/pull/606#discussion_r2261556309